### ISSUE TITLE
SSL on Entire Website

### ISSUE THREAD

**Issue by sgerritse** [MastermindMedia/MCAProTools#48]
We want the entire website to be secure as possible & follow our merchants instructions. 

![screen shot 2017-04-01 at 6 26 34 pm](https://cloud.githubusercontent.com/assets/10511034/24582907/e3c3c1e8-1708-11e7-96f1-ac4d456d5130.png)

...Currently, website loads insecurely, and does not redirect to HTTPS always. (Heres a little checklist for you to use as reference.)

![screen shot 2017-04-01 at 6 30 20 pm](https://cloud.githubusercontent.com/assets/10511034/24582935/9e5a68a4-1709-11e7-8c96-28408ed3cc1b.png)

**Comment by tarundhanwani** [comment]
Yup got it, I know what to do. Question, has the SSL been purchased and the certificate installed on the webhost/domain?

I'll start working on these open tickets on Tuesday  once I get back.

Thanks

**Comment by sgerritse** [comment]
I have Sumeena working on this task. You need not worry about the SSL.

**Comment by sgerritse** [comment]
Just keep this in mind, and review website for anymore issues related to SSL.
